include/shark/Algorithms/DirectSearch/Indicators/LeastContributorApproximator.h

```diff
@@ -393,94 +224,31 @@ struct LeastContributorApproximator {
 
 	}
 
-	/**
```

Comment:
Okay, you I will ignore that this is commented out since you plan to look at this soon anyway.